backend/src/test/java/gov/cdc/usds/simplereport/service/DeviceTypeServiceTest.java

```diff
@@ -103,6 +109,10 @@ void createAndDeleteDeviceTypes_adminUser_success() {
     // GIVEN
     SpecimenType swab1 = specimenTypeRepository.save(new SpecimenType("Hair", "000111222"));
     SpecimenType swab2 = specimenTypeRepository.save(new SpecimenType("Mouth", "112233445"));
+    SupportedDisease disease1 =
+        supportedDiseaseRepository.save(new SupportedDisease("Disease1", "D1"));
+    SupportedDisease disease2 =
```

Comment:
nit: you should be able to read out the real diseases here if you prefer .

Reply:
Good call, didn't think about this when rebasing, will fix!

Reply:
Strangely, calling `SupportedDisease disease1 =  supportedDiseaseRepository.findSupportedDiseaseByNameContains("COVID");` seems to return `null` 🤔 Shouldn't the diseases be loaded since this test extends BaseServiceTest?